superset/assets/visualizations/nvd3_vis.js

```diff
@@ -195,6 +195,33 @@ function nvd3Vis(slice) {
             chart.labelThreshold(0.05)  // Configure the minimum slice size for labels to show up
               .labelType(fd.pie_label_type);
             chart.cornerRadius(true);
+
+            // Configure the pie more tyles
+            if (fd.pie_half_circle){
+              chart.pie
+              .startAngle(function(d) { return d.startAngle/2 -Math.PI/2 })
+              .endAngle(function(d) { return d.endAngle/2 -Math.PI/2 });
+            } 
+
+            if (fd.pie_title){
```

Comment:
I don't see references to pie_title, pie_title_pffset and pie_pad_angle anywhere else. Am i missing something?